Marlin/Marlin_main.cpp

```diff
@@ -2383,14 +2370,10 @@ static void clean_up_after_endstop_or_probe_move() {
 
     float measured_z = NAN;
     if (!DEPLOY_PROBE()) {
-      measured_z = run_z_probe(
-        #if HAS_CALIBRATION_PROBE
-          is_calibration
-        #endif
-      ) + zprobe_zoffset;
+      measured_z = run_z_probe(unprotected) + zprobe_zoffset;
```

Comment:
always calls run_z_probe with unprotected parameter; it is defined in all cases;